### PR TITLE
76445: added endpoint to edit a single widget configuration

### PR DESCRIPTION
**Pull request by theo-ritense**

### CODE REVIEW

File: dashboard/src/test/kotlin/com/ritense/dashboard/web/rest/AdminDashboardResourceIT.kt [hunk 1]
```diff
@@ -313,20 +261,37 @@ class AdminDashboardResourceIT : BaseIntegrationTest() {
...
     @Test
-    fun `should get widget configuration by id`() {
+    fun `should update widget configuration`() {
         val dashboard = dashboardService.createDashboard("Test dashboard", "Test description")
-        widgetConfigurationRepository.save(
-            WidgetConfiguration(
-                key = "doorlooptijd",
+        createWidgetConfiguration(dashboard)
+
+        val udpateRequest = SingleWidgetConfigurationUpdateRequestDto(
```
Comment (marijnritense) [inline dashboard/src/test/kotlin/com/ritense/dashboard/web/rest/AdminDashboardResourceIT.kt:268]:
```suggestion
        val updateRequest = SingleWidgetConfigurationUpdateRequestDto(
```

File: dashboard/src/test/kotlin/com/ritense/dashboard/web/rest/AdminDashboardResourceIT.kt [hunk 2]
```diff
@@ -313,20 +261,37 @@ class AdminDashboardResourceIT : BaseIntegrationTest() {
...
+                dataSourceKey = "doorlooptijd2",
+                dataSourceProperties = jacksonObjectMapper().readTree("""{ "threshold": 500 }""") as ObjectNode,
+                displayType = "donut",
+                displayTypeProperties = jacksonObjectMapper().readTree("""{ "useKpi": false }""") as ObjectNode,
             )
+
+        mockMvc.perform(
+            put("/api/management/v1/dashboard/{dashboardKey}/widget-configuration/{widgetKey}", "test_dashboard", "doorlooptijd")
+                .contentType(APPLICATION_JSON_VALUE)
+                .content(jacksonObjectMapper().writeValueAsString(udpateRequest))
```
Comment (marijnritense) [inline dashboard/src/test/kotlin/com/ritense/dashboard/web/rest/AdminDashboardResourceIT.kt:279]:
```suggestion
                .content(jacksonObjectMapper().writeValueAsString(updateRequest))
```

File: dashboard/src/test/kotlin/com/ritense/dashboard/web/rest/AdminDashboardResourceIT.kt (hunk 1 above)
Comment (theo-ritense) [reply dashboard/src/test/kotlin/com/ritense/dashboard/web/rest/AdminDashboardResourceIT.kt]: fixed

File: dashboard/src/test/kotlin/com/ritense/dashboard/web/rest/AdminDashboardResourceIT.kt (hunk 2 above)
Comment (theo-ritense) [reply dashboard/src/test/kotlin/com/ritense/dashboard/web/rest/AdminDashboardResourceIT.kt]: fixed

**Review by marijnritense — APPROVED** [review]

**Pull request merged**